docs/helper.md

```diff
@@ -0,0 +1,94 @@
+# `gpii.handlebars.helper`
+
+This is the base grade for Fluid components that are intended to add new [Handlebars block helpers](http://handlebarsjs.com/block_helpers.html)
+to an instance of [`gpii.express.hb`](handlebars.md).
+
+## Component Invokers
+
+### `{that}.getHelper()`
+
+Each "helper" module is expected to implement a `getHelper` invoker with an invoker that returns a helper function. something like the following:
+
+```
+fluid.registerNamespace("your.helper");
+
+your.helper.getHelperFn = function(that){
+   return function(arg1, arg2) {
```

Comment:
Given that I am the only one who has written these to date, it is fairly low risk.  I believe I'd just need to update this package and the "deps" helper in the sandbox.